prometheus_hardware_exporter/collectors/redfish.py

```diff
@@ -221,7 +221,29 @@ def _storage_root_uri(cls, system_id: str, storage_name: str) -> str:
           system_id: "S1"
           storage_name" "Storage"
         """
-        return cls.systems_root_uri + f"{system_id}/{storage_name}/"
+        return cls.systems_root_uri + f"{system_id}/{storage_name}"
+
+    def _find_storage_controller_data(self, storage_controllers: Dict) -> List[Dict]:
+        """Return storage controller list.
+
+        Returns:
+            storage_controller_list: list of storage controllers for 
+            requested storage id.
+
+        """
+        if "StorageControllers" in storage_controllers:
+            storage_controllers_list: List[Dict] = storage_controllers.get("StorageControllers")
+        elif "Controllers" in storage_controllers:
```

Comment:
docs: Could we leave a comment here mentioning about the ILO version/ vendor that requires the code to handle the "Controllers" key?